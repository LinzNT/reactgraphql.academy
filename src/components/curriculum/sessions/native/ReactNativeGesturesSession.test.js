import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import ReactNativeGesturesSession from './ReactNativeGesturesSession'

describe('<ReactNativeGesturesSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<ReactNativeGesturesSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
