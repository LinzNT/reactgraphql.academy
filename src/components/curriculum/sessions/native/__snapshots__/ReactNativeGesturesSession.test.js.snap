// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactNativeGesturesSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      JSResponder
    </styled.li>
    <styled.li>
      PanGestureHandler
    </styled.li>
    <styled.li>
      RotationGestureHandler
    </styled.li>
    <styled.li>
      PinchGestureHandler
    </styled.li>
    <styled.li>
      Gestures Lifecycle
    </styled.li>
    <styled.li>
      Composing Handlers
    </styled.li>
    <styled.li>
      react-native-gesture-handler limitations
    </styled.li>
  </styled.ul>
</Session>
`;
